app/models/minion.rb

```diff
@@ -54,6 +63,18 @@ def self.assign_roles!(roles: {}, default_role: :minion)
     end
   end
 
+  def self.select_masters(roles)
```

Comment:
That would make sense for me as it is right now. I don't know if in the future we'd have other roles though. So if you go for this solution, write in a documentation comment this assumption 😉